tests/run-server.sh

```diff
@@ -100,6 +100,11 @@ if test -z "$IP";then
 	echo "Detected IP is null!"
 	stop
 fi
-echo "$IP"
+
+serv_ip=$( echo $IP | cut -d' ' -f1 )
```

Comment:
is this related?

Reply:
Currently, we do not clearly show the Server IP address after starting the docker. 

Hence, made the changes to show the Server IP and command to start the tests clearly.

Let me know your views.

Reply:
ok